feat: support multiple schema (#714)
SAP/cloud-sdk-js: CHANGELOG.md

- Batch `ReadResponse.as` does not parse errors anymore, but throws an error if the response in fact was an `ErrorResponse`. To avoid this check `response.isSuccess()` before casting.
- Rename deprecated ODataRequestConfigs e.g. `ODataCreateRequestConfig` -> `ODataCreateRequestConfigLegacy` due to naming conflicts.

> - [Generator] In order to support multi-schema generation, the edmx schema/namespace related types are adjusted.

Reviewer: ```suggestion
- [Generator] In order to support multi-schema generation, the edmx schema/namespace related types have been extended.
```
